ci/scripts/license_diff.py

```
@@ -73,73 +45,118 @@ def main(base_branch: str) -> None:
+
+        if any(str(name).startswith(prefix) for prefix in FILTERED_PACKAGE_PREFIXES):
+            continue
+
+        head_variants: dict[tuple[str, str], Package] = {package_variant_key(pkg): pkg for pkg in head_pkgs}
+        base_variants: dict[tuple[str, str], Package] = {package_variant_key(pkg): pkg for pkg in base_pkgs}
+
+        added: set[tuple[str, str]] = set(head_variants.keys()) - set(base_variants.keys())
+        removed: set[tuple[str, str]] = set(base_variants.keys()) - set(head_variants.keys())
+
+        if added and removed and len(added) == len(removed):
+            for b, h in zip(removed, added):
```

Ruff (0.15.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)